files/about.html

```diff
@@ -384,7 +384,7 @@ <h2 class="team-member-name">
                                 </h2>
 
                                 <h3 class="team-member-name">
-                                    
+                                    Michell Argus 
```

Comment:
Hey Lucas, for Michell's information section, let's create a separate pr for her so we can merge that as soon as possible. @lucasstranger1

Reply:
Let's keep this pr related to the carousel only. Thanks in advance

Reply:
I just saw the comment.
I will try to create another branch that only changes carousel features. I think I accidentally touched the original CSS code.

Reply:
Sounds good